codacy/values.yaml

```diff
@@ -281,11 +281,13 @@ fluentdoperator:
   rbac:
     create: true
   useReleaseNamespaceOnly: true
-  # Doesn't work:
-  # https://github.com/fluent/fluentd-kubernetes-daemonset#disable-systemd-input
+
   fluentd:
     extraEnv:
+      # https://github.com/fluent/fluentd-kubernetes-daemonset#disable-systemd-input
       FLUENTD_SYSTEMD_CONF: disable
+      # https://github.com/fluent/fluentd-kubernetes-daemonset#disable-prometheus-input-plugins
+      #FLUENTD_PROMETHEUS_CONF: disable
```

Comment:
This is enabled by default. I just left it commented out so we can refer back to this if we want to disable fluentd prometheus monitoring in the future.